src/repository/opamRepositoryBackend.mli

```diff
@@ -53,7 +54,7 @@ module type S = sig
       verifications. The file or directory returned is always temporary and
       should be cleaned up by the caller. *)
   val fetch_repo_update:
-    repository_name -> ?cache_dir:dirname -> dirname -> url ->
+    repository_name -> ?full_fetch:bool -> ?cache_dir:dirname -> dirname -> url ->
```

Comment:
Why is it needed for `fetch_repo_update`?

Reply:
It's not needed, don't know I missed this.